lib/runner.go

```diff
@@ -78,9 +73,6 @@ func (r *Runner) RunCommand() error {
 	}
 	if !r.options.JSONOutput {
 		fmt.Fprintf(r.stdout, "Found %d test files\n", len(testFiles))
-		if !r.options.InOrder {
-			fmt.Fprintf(r.stdout, "Using seed: %d\n", r.options.RandomSeed)
-		}
```

Comment:
This change I disagree with.

The information can be needed to reproduce issues.

While most of the time a test case issue is just something within the test case itself, it is possible to be an ordering issue where some other test did not clean up properly, causing the issue now, much later.

To reproduce such issues we must have the capability to force the test brain to run the tests in the same order as when the issue appeared. This is controlled by the seed. Part one of this capability is therefore `--seed` option to set it. Part two is actually knowing the seed used by a test run. The removal above destroys the part of the brain giving us this knowledge.

I am ok with moving the output to somewhere else. I am not ok with removing it entirely.

Reply:
I see. I'll bring it back, then.

Reply:
Thank you.